Modules/Loadable/Volumes/SubjectHierarchyPlugins/qSlicerSubjectHierarchyVolumesPlugin.cxx

```diff
@@ -484,13 +484,13 @@ QString qSlicerSubjectHierarchyVolumesPlugin::tooltip(vtkIdType itemID)const
     imageData->GetDimensions(dimensions);
     double spacing[3] = {0.0,0.0,0.0};
     volumeNode->GetSpacing(spacing);
-    tooltipString.append( QString(" (Dimensions: %1x%2x%3  Spacing: %4mm x %5mm x %6mm)")
+    tooltipString.append( QString(" (" + tr("Dimensions:") + " %1x%2x%3  " + tr("Spacing:") + " %4mm x %5mm x %6mm)")
```

Comment:
This is a bit hard to read. Please add `tr("Dimensions:")` and `tr("Spacing")` by using `.arg()`

It would look something like:
    tooltipString.append( QString(" (%1 %2x%3x%4  %5 %6mm x %7mm x %8mm)")
```